ledger-core/application/testutils/launchnet/launchnet.go

```diff
@@ -209,71 +228,94 @@ func stopInsolard(cmd *exec.Cmd) error {
 	return nil
 }
 
-func waitForNetworkState(cfg appConfig, state network.State) error {
+type outputWriter struct {
+	skip bool
+}
+
+func (ow outputWriter) log(a ...interface{}) {
+	if !ow.skip {
+		fmt.Println(a...)
+	}
+}
+
+func waitForNetworkState(cfg appConfig, state network.State, verbose bool) error {
```

Comment:
I'd rather have launchnet.SetVerbose(false), then passing verbose to every function. I dont think we will need to mix verbose in different functions

Reply:
done